shuffler.js

```diff
@@ -48,6 +48,76 @@ class Shuffler {
         return this.versions[versionIndex];
     }
 
+    /**
+     * Version to measure in a benchmark
+     * against {@link randomWithoutReplacement}
+     * @param {Number} count
+     */
+    randomWithoutReplacementUsingFilter(count) {
+        if (this.versions.length === 0 || this.positions.length === 0) {
+            return null;
+        }
+
+        if (this.versions.length <= count) {
+            return this.versions;
```

Comment:
Maybe, it makes sense to do a shuffle on what we are returning here (in case we have more than 1 element)